fuzz/fuzz_targets/outpoint_string.rs

```diff
@@ -0,0 +1,88 @@
+
+extern crate bitcoin;
+
+use bitcoin::blockdata::transaction::OutPoint;
+use bitcoin::consensus::encode;
+
+use std::str::FromStr;
+
+fn do_test(data: &[u8]) {
+    let lowercase: Vec<u8> = data.iter().map(|c| match *c {
```

Comment:
Maybe just want to do the lowercase conversion on the assert_eq checks instead of on the input?